.github/workflows/e2e-tests.yml

```diff
@@ -55,5 +55,5 @@ jobs:
       - name: Cypress Run (Production Backend)
         run: pnpm run test:e2e-real:start
 
-      - name: Cypress Run (Mock Backend)
-        run: pnpm run test:e2e-mock:start
+#      - name: Cypress Run (Mock Backend)
+#        run: pnpm run test:e2e-mock:start
```

Comment:
The mock service seems to be timing out and not running. We should look at this in the future or determine we don't need a mock backend e2e test.

https://github.com/FormidableLabs/nextjs-sanity-fe/actions/runs/4813949770/jobs/8571899972